fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -181,6 +183,20 @@ class BrowserToolbarView(
                     interactor = interactor,
                 )
             }
+
+            browserFragmentStore = BrowserFragmentStore(
+                initialState = BrowserFragmentState(),
+                middleware = listOf(ToolbarCfrMiddleware(
+                    components.core.store,
+                    view,
+                    components.settings,
+                    DefaultShoppingExperienceFeature(),
+                    customTabSession?.id,
+                    {},
+                    {},
+                    {},
```

Comment:
I think there is opportunity to improvise a bit in cases like this - but the easiest case is defining the CFR callbacks inline in the view unless there is a strong desire to unit test them.